components/StatusError/index.js

```diff
@@ -54,7 +48,7 @@ export default compose(
     skip: props => props.statusCode !== 404 || !props.router.asPath,
     options: ({ router: { asPath } }) => ({
       variables: {
-        path: cleanAsPath(asPath)
+        path: asPath
```

Comment:
```suggestion
        path: asPath.split('#')[0]
```

Reply:
> Clients are not supposed to send URI-fragments to servers when they retrieve a document, and without help from a local application (see below) fragments do not participate in HTTP redirections.
[Fragment identifier on Wikipedia](https://en.wikipedia.org/wiki/Fragment_identifier)

I'd follow that and also avoid sending fragment up here.